Src/DependencyCollector/Shared.Tests/Implementation/ProfilerSqlProcessingTest.cs

```diff
@@ -361,6 +366,280 @@ public void OnEndLogsWarningWhenObjectPassedToEndWithoutCorrespondingBeginAsync(
             }
         }
 
+        [TestMethod]
+        public void FieldsForSqlConnectionAreCollectedCorrectly()
+        {
+            SqlConnection connection = GetSqlConnectionTest();
+            string actualResourceName = this.sqlConnectionProcessingProfiler.GetResourceName(connection);
+            Assert.AreEqual(ExpectedTarget, actualResourceName);
+
+            string actualTargetName = this.sqlConnectionProcessingProfiler.GetResourceTarget(connection);
+            Assert.AreEqual(ExpectedTarget, actualTargetName);
+
+            string actualCommandText = this.sqlConnectionProcessingProfiler.GetCommandName(connection);
+            Assert.AreEqual(ExpectedCommandTextForSqlConnection, actualCommandText);
+        }
+
+        [TestMethod]
+        public void OnEndAsyncSendsCorrectTelemetry_1ArgumentOverride()
+        {
+            var command = GetSqlCommandTestForStoredProc();
```

Comment:
method name is self descriptive, and it's test.

---
In reply to: [98072442](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/298#discussion_r98072442) [](ancestors = 98072442)